packages/cli/src/lib/yargs-cli.integration.test.ts

```diff
@@ -16,11 +16,11 @@ describe('yargsCli', () => {
     expect(parsedArgv.progress).toBe(true);
   });
 
-  it('should provide a correct default value for the config', async () => {
+  it('should provide no default value for the config', async () => {
```

Comment:
```suggestion
```
If the logic has been moved to config-middleware, the test case here is misleading - the config does have a default prefix value but after reading this test case, it seems that one always has to provide `--config` argument. Please add a test to `config-middleware` instead.

Reply:
done